elyra/pipeline/processor.py

```diff
@@ -424,6 +424,9 @@ def _process_dictionary_value(self, value: str) -> Union[Dict, str]:
             except Exception:
                 pass
 
+        if not value or value == "None":
+            return {}
+
```

Comment:
Here is too late, you will get errors on line 420 when you try to strip to None, no? I would move this to top of the function.

Reply:
btw, when do we get "None" and what is the difference from just using None?

Reply:
Ha we are on the exact same page based on the commit I just pushed. I have the same sort of question as to whether we should allow `"None"` to evaluate as `None` or keep the empty value. I've tested it with pipeline submission on Airflow so far and nothing has changed from the prior commit. As in, the pipeline executes successfully regardless.

Reply:
We would get `"None"` in the case that the user types None into the parameter field. And the description for some parameters mentions that None is an option, e.g. the BashOperator `env` parameter description says "If env is not None, ...". I feel like this may indicate to some users that they should/could type None into that field and expect the value of `None` to show up in the DAG.

Reply:
Hmm unfortunately this doesn't work very well with KFP. I'm going to just revert to only using the empty list or dictionary in both cases/ But keeping the new test to check against the "None" string